packages/storage/src/common/constants.ts

```diff
@@ -0,0 +1,2 @@
+export const DEFAULT_API_KEY =
+  "fb52f0ca630bdd02f0d5210d9e8e3568272ac765dd0709272918eefb29d3ac9e";
```

Comment:
remember to add this key to https://www.notion.so/thirdweb/Universal-RPC-API-keys-4634982244164fc4a51962e9e1ae3861